### ISSUE TITLE
Call for future maintainers

### ISSUE THREAD

**Issue by mavogel** [terraform-providers/terraform-provider-docker#306]
Hey folks,

finally, I had the time to migrate this community provider to the org I recently joined => https://github.com/kreuzwerker/terraform-provider-docker

I am almost done with the releasing process to the [registry](https://registry.terraform.io/) and for the future, I'd really love to have more maintainers for this provider :) 

So based on previous conversations and PRs: Do you want to be a future maintainer of this provider?

- @mkeeler
- @dubo-dubon-duponey
- @suzuki-shunsuke
- @sergei-ivanov
- @edgarpoce

//@mkeeler as you are from the HashiCorp org: how would be the process of accepting the Contribution guidelines? Did I forget something?

Best Manuel

**Comment by dubo-dubon-duponey** [comment]
Hi Manuel,

Really appreciate the offer.
Though, I do not think I have the available bandwidth to engage seriously here, so... maybe another time.

Best regards,

- O.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by suzuki-shunsuke** [comment]
Hi Manuel,

Thank you for your offer.
I'm interested in the co-maintainer.

By the way, why is this provider migrated?

**Comment by mavogel** [comment]
Great to hear that @suzuki-shunsuke  👍 The provider was migrated because as of `terraform 0.13` the provider still work from the deprecate [releases](https://releases.hashicorp.com/terraform-provider-docker/) but also from the new [registry](https://registry.terraform.io/). For the future all providers shall be migrated to the new registry. See [here](https://www.hashicorp.com/blog/announcing-hashicorp-terraform-0-13) for details. HTH

**Comment by suzuki-shunsuke** [comment]
Thank you for your explanation.
I understood the change from v0.12 to v0.13.

main.tf

```hcl
provider "docker" {
}

resource "docker_image" "alpine" {
  name = "alpine:3.10.3"
}
```

v0.12

`terraform init` is succeeded.

```
$ terraform version
Terraform v0.12.29

$ terraform init

Initializing the backend...

Initializing provider plugins...
- Checking for available provider plugins...
- Downloading plugin for provider "docker" (terraform-providers/docker) 2.7.2...
...
```

v0.13

`terraform init` is failed.

```
$ terraform version
Terraform v0.13.5

$ terraform init

Initializing the backend...

Initializing provider plugins...
- Finding latest version of hashicorp/docker...

Error: Failed to install providers

Could not find required providers, but found possible alternatives:

  hashicorp/docker -> terraform-providers/docker

If these suggestions look correct, upgrade your configuration with the
following command:
    terraform 0.13upgrade .
```

**Comment by mavogel** [comment]
Yes, that's right!  There is still a migration to `0.13` pending of the provider.

**Comment by edgarpoce** [comment]
Hi Manuel, many thanks for the invitation. Unfortunately I don't have
the bandwidth these days. I'd love to help later, once the covid new
normal ends, particularly the part where the kids stay at home always
:).

Kind regards,
Edgar

**Comment by mkeeler** [comment]
Hi Manuel,

Is there a new CLA or something after migration of the provider to the
kreuzwerker org that I would need to sign? Is that what you meant by "what
would that process look like" as I would have to run it by legal at
HashiCorp.

As for maintainership, unfortunately I also don’t have the spare cycles to
devote to it right now either.

-- 
Matt Keeler

-- 
Matt Keeler

On November 11, 2020 at 10:25:44 AM, Manuel Vogel (notifications@github.com)
wrote:

Hey folks,

finally, I had the time to migrate this community provider to the org I
recently joined => https://github.com/kreuzwerker/terraform-provider-docker

I am almost done with the releasing process to the registry
<https://registry.terraform.io/> and for the future, I'd really love to
have more maintainers for this provider :)

So based on previous conversations and PRs: Do you want to be a future
maintainer of this provider?

   - @mkeeler <https://github.com/mkeeler>
   - @dubo-dubon-duponey <https://github.com/dubo-dubon-duponey>
   - @suzuki-shunsuke <https://github.com/suzuki-shunsuke>
   - @sergei-ivanov <https://github.com/sergei-ivanov>
   - @edgarpoce <https://github.com/edgarpoce>

//@mkeeler <https://github.com/mkeeler> as you are from the HashiCorp org:
how would be the process of accepting the Contribution guidelines? Did I
forget something?

Best Manuel

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/terraform-providers/terraform-provider-docker/issues/306>,
or unsubscribe

**Comment by mavogel** [comment]
Good point @mkeeler, thanks for pointing out. I wrote a mail to provider-dev at HashiCorp to ask for the right steps. 
A pity you cannot join as a maintainer. Collaborations are always welcome.
